### DISCUSSION TITLE
Can i dynamic unload some object & map block ?

### DISCUSSION THREAD

**Discussion by Lyoko-Jeremie** [enable3d/enable3d#278] [Help / Questions]
then i do some test , i perceive that the phyics engine will broken when create very large map or many many triangle object .

can i dynamic unload some object&map block from scene that out of view to save phyics engine , then load it after player back ? 

or just i use it in wrong way ?

**Comment by yandeu** [comment]
Yes, you can manually check if an object is out of view and remove the physics from that object. To skip an object from visual rendering, use [visible](https://threejs.org/docs/#api/en/core/Object3D.visible).

**Reply by Lyoko-Jeremie** [reply]
i mean , how to remove the physics from that object ?

**Reply by yandeu** [reply]
https://enable3d.io/examples/create-and-destroy-objects-and-physics.html
